File: client/src/app/classes/message.ts
export class Message {
    messageTime: string;
    constructor(public message: string, public messageUser: string) {
        this.messageTime =
            new Date().getHours().toString().padStart(2, '0') +
            ':' +
            new Date().getMinutes().toString().padStart(2, '0') +
            ':' +
            new Date().getSeconds().toString().padStart(2, '0');
    }
}

File: prototypeLeger/app/src/main/java/com/example/scrabbleprototype/model/Message.kt
package com.example.scrabbleprototype.model

import java.util.*
import kotlinx.serialization.Serializable
import java.text.SimpleDateFormat

@Serializable
class Message(var message: String, var messageUser: String) {

    var messageTime: String = getTimestamp()

    fun getTimestamp(): String {
        val time = Date().time
        val timestampFormat = SimpleDateFormat("HH:mm:ss", Locale.getDefault())
        return timestampFormat.format(time)
    }
}
